clients/README.md

```diff
@@ -86,6 +88,18 @@ subxt metadata -f bytes > runtime/metadata-{your-chain-name}.scale
 subxt metadata -f bytes --url http://{chain-url} > runtime/metadata-{your-chain-name}.scale
 ```
 
+After making the changes, the first step is to run the local node.
+
+```bash
+cargo run --bin spacewalk-standalone --release -- --dev
+```
+
+Once the local node is running, run this command:
+
+```bash
+subxt metadata -f bytes > runtime/metadata-{your-chain-name}.scale
```

Comment:
It will always be the standalone one with the local node, so we can make it explicit in this command.
```suggestion
subxt metadata -f bytes > runtime/metadata-standalone.scale
```